README.rst

```diff
@@ -211,14 +211,50 @@ actually accepts. Read more about it in the `HTML 5 Rocks CORS tutorial
 If ``True``, cookies will be allowed to be included in cross-site HTTP
 requests. Defaults to ``False``.
 
+``CORS_MODEL``
+~~~~~~~~~~~~~~
+
+If set, this should be the path to a model to look up allowed origins, in the
+form ``app.modelname``. Defaults to ``None``.
+
+The model should inherit from ``corsheaders.models.AbstractCorsModel`` and specify
+the allowed origin in the ``CharField`` called ``cors``.
+
+CSRF Integration
+----------------
+
+Most sites will need to take advantage of the `Cross-Site Request Forgery
+protection <https://docs.djangoproject.com/en/dev/ref/csrf/>`_ that Django
+offers. CORS and CSRF are distinct concepts, and Django will not use your CORS
+configuration to exempt sites from the ``Referer`` checking that it does on
+secure requests. The way to do that is with the `CSRF_TRUSTED_ORIGINS setting
+<https://docs.djangoproject.com/en/dev/ref/settings/#csrf-trusted-origins>`_.
+For example:
+
+.. code-block:: python
+
+    CORS_ORIGIN_WHITELIST = (
+        'read.only.com',
```

Comment:
Well, note that the first example domain on this page is `google.com`. 😏 And the Django documentation includes things like `thebeatles.com`, etc. I don't see any harm in using a real domain since it's not actually a link.

Also, that restriction seems out of place for a project that is explicitly concerned with listing different domains. Listing a bunch of subdomains of `example.com` would be very confusing, since subdomains are different from full domains in a number of important ways.

My goal here was to take the opportunity to provide some hints on the distinction between the two settings. Let me know, though, if you have a preferred alternative.